app/javascript/components/search/controls/download/DownloadSelectionModal.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react'
 import Modal from 'react-bootstrap/lib/Modal'
 import _cloneDeep from 'lodash/cloneDeep'
+import { partition } from 'lodash'
```

Comment:
```suggestion
import _partition from 'lodash/partition'
```
The underscore prefix makes it clearer that this is a lodash method, and also utilizes lodash's partial import capability (so the import only grabs exactly the lodash fragment you need)

Reply:
39fe8e0